src/commands/frameworks-stacks-list.ts

```diff
@@ -4,27 +4,39 @@ import { needProjectId } from "../projectUtils";
 import * as gcp from "../gcp/frameworks";
 import { FirebaseError } from "../error";
 import { logger } from "../logger";
+const Table = require("cli-table");
 
 export const command = new Command("backends:list")
   .description("List backends of a Firebase project.")
   .option("-l, --location <location>", "App Backend location", "us-central1")
   .action(async (options: Options) => {
     const projectId = needProjectId(options);
     const location = options.location as string;
+    const table = new Table({
+      head: ["Backend Id", "Repository Name", "URL", "Location", "Created Date", "Updated Date"],
+      style: { head: ["yellow"] },
+    });
 
-    let backends;
+    let backendsList;
     try {
-      backends = await gcp.listBackend(projectId, location);
-      /**
-       * TODO print this in a prettier way.
-       */
-      logger.info(backends);
+      backendsList = await gcp.listBackend(projectId, location);
```

Comment:
This is kind of weird imo that GCP exposes it like this without the plural form, which goes against AIP best practices https://google.aip.dev/132, specifically "The remainder of the RPC name should be the plural form of the resource being listed."

Nothing we can do I guess just wanted to call this out